src/ypd/templates/SubmissionPage.html

```diff
@@ -2,36 +2,75 @@
 <html>
 	<head>
-		<title>Project Submission</title>
+		<title>YCP Project Database:Project Submission</title>
+        <link rel="icon" href="Logo.png" type="image/png">
+        <link href="https://fonts.googleapis.com/css?family=Alfa+Slab+One&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css?family=Rokkitt&display=swap" rel="stylesheet">
+        <link href="../static/TeamProjectStyleSheet.css" rel="stylesheet" type="text/css">
 	</head>
 
 	<body>
-	<div>
-		<h1>Project Submission</h1>
-	</div>
-		<form action="/submission" method="POST">
-			
-			<fieldset>
-				<p>
-					<label>Project Title:</label>
-					{{ form.title}}
-				</p>
-				<p>
-					<label>Project Summary:</label>
-					{{form.description}}
-				</p>
-				<p>
-					<label>Poster:</label>
-					{{form.poster}}
-				</p>
-				<p>
-					{{form.hidden_tag()}}
-					{{form.projType}}
-				</p>
-				<p>
-				<input type = "submit" value = "Go" >
-				</p>
-			</fieldset>
-		</form>
+        <div class="mainTitle">
+            <div class="logo">
+                <img src="../static/Logo.png" alt="Logo" width="30" height="35">
+            </div>
+        
+            <div class="mainTitleText">
+                <center>YCP Project Database</center>
+            </div>    
+        
+            <div class="dropDown">
+                <div class="menuTab">
+                    <center>
+                        <div class="bar"></div>
+                        <div class="bar"></div>
+                        <div class="bar"></div>
+                    </center>
+                </div>
+                <div class="dropDown-Content">
+                    <a href="{{ url_for('IndexView:get') }}">Project Listing</a>
+                    <a href="{{ url_for('SubmissionView:post') }}">Post A Project</a>
+                    <a href="{{ url_for( 'UserView:login') }}">Login/Sign Up</a>
+                </div>
+            </div>
+        </div>
+
+        <div class="searchBar">
+            <form action="/search">
+                <input type="text" name="search_text" size="12" placeholder="Search.." value="{{search_text}}" />
+                <input type="submit" value="Search">
+            </form>
+        </div>
+        
+        <div class="pageHeader">
+            <h1>Project Submission</h1>
+        </div>
+        
+        <form action="/submission" method="POST">	
+            <div class="submissionBody">
+                <fieldset>
+                    <p>
+                        <label>Project Title:</label><br/>
+                        {{ form.title(style="width: 150px")}}
+                    </p>
+                    <p>
+                        <label>Project Summary:</label><br/>
+                        {{form.description(style="width: 75%")}}
+                    </p>
+                    <p>
+                        <label>Poster:</label><br/>
+                        {{form.poster}}
```

Comment:
We shouldn't call this poster. Something like project type would be more appropriate